### DISCUSSION TITLE
Removing homematicip_local from HAOS which is not shown in the integration page

### DISCUSSION THREAD

**Discussion by carusoe** [SukramJ/aiohomematic#1442] [Q&A]
Hi,

I have an issue with my HAOS installation. In the core log appears the following error:

`2024-02-20 02:22:25.344 ERROR (MainThread) [homeassistant.helpers.service] Failed to load integration: homematicip_local
homeassistant.loader.IntegrationNotFound: Integration 'homematicip_local' not found.
2024-02-20 02:22:25.472 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [140213927488960] Error handling message: Unknown error (unknown_error) Sandro from 192.168.179.27 (Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:122.0) Gecko/20100101 Firefox/122.0)
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/websocket_api/decorators.py", line 26, in _handle_async_response
    await func(hass, connection, msg)
  File "/usr/src/homeassistant/homeassistant/components/websocket_api/commands.py", line 484, in handle_get_services
    payload = await _async_get_all_descriptions_json(hass)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/websocket_api/commands.py", line 465, in _async_get_all_descriptions_json
    descriptions = await async_get_all_descriptions(hass)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 633, in async_get_all_descriptions
    translations = await translation.async_get_translations(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/translation.py", line 378, in async_get_translations
    return await cache.async_fetch(language, category, components)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/translation.py", line 219, in async_fetch
    await self._async_load(language, components_to_load)
  File "/usr/src/homeassistant/homeassistant/helpers/translation.py", line 248, in _async_load
    raise int_or_exc
homeassistant.loader.IntegrationNotFound: Integration 'homematicip_local' not found.`

What I did is one day in the past to try to install homematicip_local on my server by accident. Now I need to figure out how to fix this and remove the homematicip_local integration entirely.

Is there any help? I don't want to install the HA from scratch...

**Comment by SukramJ** [comment]
How did you remove the integration?

**Reply by carusoe** [reply]
Well, I have to admit the I installed the integration be accident. Let me explain:
I have two HAOS servers running that are connected with the integration "Remote Home Assistant". On the main server the updates of the client server are shown. One day it showed that an update of homematicip_local was pending on the client server. On the main server I clicked by accident on install. It showed an error - therefore, I didn't think of any bad... :-/

**Reply by carusoe** [reply]
Now I have the problem that I cannot update the main server due to this error - I think.

**Reply by SukramJ** [reply]
That doesn't answer my question.

**Reply by carusoe** [reply]
In fact, I didn't removed it since it doesn't show in the integration screen and this is exactly my question: where can I remove it when not shown in the integration screen?

**Reply by SukramJ** [reply]
Looks like your problem is related to `Remote Home Assistant`, but I never used that.
Maybe there is an entry in `/config/.storage/core.config_entries` that causes these messages, but manually (incorrectly) editing the file can kill the entire system.

**Reply by carusoe** [reply]
Found it... Checking... but before I do a backup... ;-)

**Reply by carusoe** [reply]
Hi,

thanks for your suggestion. I checked all the files containing "homematicip_local" in the `/config/.storage` - didn't find anything. 

Any other idea?

The alternative is that I start from scratch. But then I need a hint how to migrate Grafana and influxDB.

**Reply by SukramJ** [reply]
> Any other idea?

No.
I assume the folder `custom_components/homematicip_local` has been removed.

**Reply by carusoe** [reply]
Yes, this folder does not exist. I doubt that it ever existed on my main HA-server.
:-)

**Reply by SukramJ** [reply]
>  I doubt that it ever existed on my main HA-server.

So nothing to do here. Better you ask `Remote Home Assistant` for help.
